lib/extension/gantt_task_iterable_extension.dart

```diff
@@ -1,9 +1,10 @@
-import 'package:gantt_view/model/gantt_event.dart';
+import 'package:gantt_view/model/gantt_task.dart';
 
-extension GanttEventListExtension on Iterable<GanttEvent> {
+extension GanttEventIterableExtension on Iterable<GanttTask> {
+  // No difference == 1 day
   int get days => endDate.difference(startDate).inDays + 1;
 
-  int get weeks => (days ~/ 7) + 1;
+  int get weeks => (days / 7).ceil();
```

Comment:
is this called on each paint? If so, the first version is faster.

Reply:
Nope, called once when the `GanttChartLayoutData` is first constructed